Raymond-Osy/iReporter
# chore(UIpages): implement LFA feedback

**#13162405268 Implement LFA front-end feedback**
#### What does this PR do?
- This PR is to show the implemented feedback on the UI given by the LFA

#### Description of Task to be completed?
- Change color of about button on the index page
- Change dummy text to a valid text
- Make UI responsive across screen sizes
- Rename dashboard to profile
- Ensure footer sticks to the bottom
- Activate logout for authenticated pages

#### How should this be manually tested?
- Visit my hosted pages using this URL: https://raymond-osy.github.io/iReporter/UI/index.html
- Check/Test each feature listed in the description above

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- Here is the ID of the PT story: #162405268. Visit this link to view it: https://www.pivotaltracker.com/n/projects/2226946

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
